.travis.yml

```diff
@@ -42,7 +42,7 @@ install:
   - if [[ $BUILD_DOCS = true ]]; then
       python setup.py install;
       pip install -r requirements-docs.txt;
-      pip install pygments prompt_toolkit ply psutil ipykernel matplotlib;
+      pip install pygments prompt_toolkit<=2 ply psutil ipykernel matplotlib;
```

Comment:
Did you mean `<2`

Reply:
Who are you who is so wise in the ways of operators? :headdesk:

Reply:
I'm an enchanter.

![05173982650c1f7883196880e703cba1a97a53-wm](https://user-images.githubusercontent.com/320553/41015193-2dda5812-6917-11e8-9ac7-aeefd54a73ee.jpg)